download.html

```diff
@@ -35,8 +36,19 @@
 var converter = new showdown.Converter();
 var rendered = document.getElementById("installation_devel_readme_rendered");
 
-function updateInstall(branch) {
-  fetch("https://raw.githubusercontent.com/humanoid-path-planner/hpp-doc/" + branch + "/doc/instructions.md")
-    .then(r => r.text()).then(r => rendered.innerHTML = converter.makeHtml(r));
+function updateInstall(url) {
+  fetch(url).then(r => r.text()).then(r => rendered.innerHTML = converter.makeHtml(r));
 }
+
+function getParameterByName(name) {
+    name = name.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]");
+    var regex = new RegExp("[\\?&]" + name + "=([^&#]*)"),
+        results = regex.exec(location.search);
+    return results === null ? "" : decodeURIComponent(results[1].replace(/\+/g, " "));
+}
+
+var branch = getParameterByName('branch');
+document.getElementById('installation_branch').val(branch);
```

Comment:
@nim65s some time ago, you remove those lines. I am not sure why you did it and could find a note explaining it. Was it on purpose ?

Reply:
Either I found that everything was working for me without that, and I removed it on purpose, or that was a mistake I don't know how to explain

Reply:
This code makes it possible to send a link with the correct version:
http://jmirabel.github.io/hpp-doc/download.html?branch=master